app/views/posts/index.html.erb

```diff
@@ -1 +1,20 @@
-<h1>This is the post#index view</h1>
\ No newline at end of file
+<div>
+  <img src='https://unsplash.com/photos/F_-0BxGuVvo' alt="user_image">
+</div>
+
+<div>
+  <p><%= @user.name %></p>
```

Comment:
Please use `post.recent_comments` to render 5 most recent comments in the `app/views/posts/index.html.erb` page.